src/Components/Product/ProductList/ProductList.js

```diff
@@ -1,19 +1,52 @@
-import axios from "axios";
-//import { useState, useEffect } from "react";
+import { Row, Col, Spinner } from "reactstrap";
+import { useState, useEffect } from "react";
+import "./ProductList.css";
 
-axios.get('http://localhost:3002/users')
-        .then(response => {
-            console.log(response);
-        });
-const ProductList = () => {
-    
-    return (
-        <div>
+function ProductList() {
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [items, setItems] = useState([]);
 
-            <img src="phone1.png" width="150" height="225" />
-            <p className="ProductList_heading">heading</p>
-        </div>
-    );
+    useEffect(() => {
+        fetch("http://localhost:3002/users")
+            .then(res => res.json())
+            .then(
+                (result) => {
+                    setIsLoaded(true);
+                    console.log(result.employee);
+                    setItems(result.employee);
+                },
+                (error) => {
+                    setIsLoaded(true);
+                    setError(error);
+                }
+            )
+    }, [])
+
+    if (error) {
+        return <div>Error: {error.message}</div>;
+    } else if (!isLoaded) {
+        return <div className="spinner">
+      <Spinner color="danger" />
+        </div>;
+    } else {
+        return (
+            <div>
+                {items.map((user) => (
+                    <div className="container">
+                        <Row className="productList">
+                            <div key={user.id}></div>
+                            <Col><img src="phone1.png" width="150" height="225" /></Col>
+                            <Col><p className="ProductList_heading">{user.name}</p>
+                                <p className="ProductList_details">{user.brand}<br />{user.RAM}<br />{user.ROM}<br />{user.Expandable}</p>
```

Comment:
could have destructured user objects and keep variable name in camelCase format.